test/unit/utils/dockerSpec.js

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import path from 'path';
+import * as path from 'path';
```

Comment:
I would be against having to do default imports like this, and keeping it like this instead:
```suggestion
import path from 'path';
```